### DISCUSSION TITLE
Getting tokens using postman

### DISCUSSION THREAD

**Discussion by nyvelius** [ory/hydra#2361] [Show and tell]
I was just testing an API using Postman and had to figure out how to get Postman working with Hydra / OAuth2. It took me a while to figure out, but it's actually pretty straightforward. Here's a quick guide in case anyone else is wondering.

1. Create a client using either the ORY Hydra CLI, REST API or one of the SDK:s. Postman will use `https://oauth.pstmn.io/v1/callback` as a callback url, so use this for any url:s set for the client. Be sure to include a client secret.

2. In Postman, fill in the chosen client id and secret. Use `https://oauth.pstmn.io/v1/callback` as a callback url. The auth and access token url:s are `https://<HYDRA>/oauth2/auth` and `.../oauth2/token`. State can be any long string.  

3. That's it, press "Get New Access Token" and login as usual.

**Comment by vinckr** [comment]
Thanks a lot for this write up!
